elide-datastore/elide-datastore-aggregation/src/test/java/com/yahoo/elide/datastores/aggregation/AggregationDataStoreIntegrationTest.java

```diff
@@ -477,6 +477,36 @@ public void aggregationComputedMetricTest() throws Exception {
         runQueryWithExpectedResult(graphQLRequest, expected);
     }
 
+    @Test
+    public void timeGrainAggregationTest() throws Exception {
```

Comment:
I think we can add more records into the test data and do both daily and monthly aggregation.

Reply:
The point of the IT tests is more to test the integration of the API with the backend.  If we need more tests - they belong in the SQLQueryEngineTest.  I did both daily and monthly tests there.  Do you feel like we should add more?

Reply:
I personally prefer to put more tests in IntegrationTest to make sure we are generating the expected `Query` object.

Reply:
What tests do you suggest adding to more fully test the Query object?